examples/slash.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Hikari Examples - A collection of examples for Hikari.
+#
+# To the extent possible under law, the author(s) have dedicated all copyright
+# and related and neighboring rights to this software to the public domain worldwide.
+# This software is distributed without any warranty.
+#
+# You should have received a copy of the CC0 Public Domain Dedication along with this software.
+# If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
+"""A simple bot with some simple commands."""
+
+import os
+
+import hikari
+
+bot = hikari.GatewayBot(token=os.environ["BOT_TOKEN"])
```

Comment:
```suggestion
# Set this to the guild to create the slash commands in.
# To create global commands, set to `hikari.UNDEFINED`.
#
# It is important to note that global commands can take up to an hour to show in the client
# and should only be used when "publishing" commands after they are built. For testing
# use guild based commands as they are instant. 
COMMAND_GUILD_ID = 0

bot = hikari.GatewayBot(token=os.environ["BOT_TOKEN"])
```

Reply:
Most of the time it just works to keep it as undefined. Idk if we should completely force users to make them guild commands.

Reply:
I have lately seen a lot of people wondering why their commands dont show up as soon as they run. At least like this they will get a nice error that they will have to fix by deciding to either place an ID or set it to `hikari.UNDEFINED`

Reply:
Fair enough